### PR TITLE
Add tooltip to disabled slider

### PR DESCRIPTION
**Pull request by garyluu**
For dockstore/dockstore#3052

### CODE REVIEW

File: src/app/shared/entry-wizard/entry-wizard.component.html [hunk 1]
```diff
@@ -30,6 +30,7 @@
           [name]="repo.repositoryName"
           [checked]="repo.present"
           [disabled]="!repo.canDelete && repo.present"
+          [matTooltip]="!repo.canDelete && repo.present ? 'This workflow is full so it cannot be deleted.' : ''"
```
Comment (coverbeck) [inline src/app/shared/entry-wizard/entry-wizard.component.html]: I don't understand the tool tip. What is a full workflow?

Comment (garyluu) [reply src/app/shared/entry-wizard/entry-wizard.component.html]: No idea, just wrote what Andrew put in the issue.

Comment (denis-yuen) [reply src/app/shared/entry-wizard/entry-wizard.component.html]: The description we give users is near the bottom of https://docs.dockstore.org/en/develop/getting-started/dockstore-workflows.html#create-workflow-stubs-from-github-bitbucket-and-gitlab

Comment (agduncan94) [reply src/app/shared/entry-wizard/entry-wizard.component.html]: A full workflow is simply not a stub, though in the future ideally there will be no stubs.

**Pull request merged**